test/tests/clone.js

```diff
@@ -84,54 +80,6 @@ describe("Clone", function() {
     });
   });
 
-  it("can clone into long path if opt set", function() {
-    var test = this;
-    var url = "https://github.com/nodegit/test.git";
-    var opts = {
-        fetchOpts: {
-          callbacks: {
-            certificateCheck: () => 0
-        }
-      }
-    };
-
-    fse.ensureDirSync(longClonePath);
-
-    if (process.platform === "win32") {
-      NodeGit.Libgit2.opts(NodeGit.Libgit2.OPT.SET_WINDOWS_LONGPATHS, 1);
```

Comment:
Now that we're moving to the updated longpaths support, we should make sure to update the tests to account for the changes instead of just removing them. See https://github.com/libgit2/libgit2/pull/5823 for the expected behavior.

Reply:
New tests have been implemented. they were converted from cloning tests to checkout tests to accommodate the slight change in functionality